bt_app_test:
  ros__parameters:
    bt_tree_node_model: "/home/ros/Eurobot-2025-Main/src/bt_app_test/bt_config/bt_tree_node_model.xml"
    groot_xml_config_directory: "/home/ros/Eurobot-2025-Main/src/bt_app_test/bt_config/"